### ISSUE TITLE
cgppindel vcfs have gz suffix but aren't

### ISSUE THREAD

**Issue by woook** [eastgenomics/eggd_vcf_handler_for_uranus#34]
`gzip: 2110552-21334Z0013-PB-MPD-MYE-F-EGG2_S14_L001_vs_TA2_S59_L008_tumor.flagged.opencga.vcf.gz: not in gzip format`

**Comment by jethror1** [comment]
forgot that they are already unzipped in the app due to previous bcftools things, I guess since these are all just in kilobytes its not worth compressing them all?

All the handler app output vcfs random recent run:
```
jethro@jethro-T490:/tmp/data/outputs_app/full_run$ du -sh
10M	.
jethro@jethro-T490:/tmp/data/outputs_app/full_run$ gzip ./*
jethro@jethro-T490:/tmp/data/outputs_app/full_run$ du -sh
2.9M	.
```
Hardly amazing gains so will just remove the `.gz` suffixes and continue

**Comment by jethror1** [comment]
Fixed in #36
